jdi-eyes-demo/src/test/resources/allure.properties

```diff
@@ -0,0 +1,3 @@
+allure.results.directory=target/allure-results
+#allure.link.issue.pattern=https://jira.com/browse/{}
```

Comment:
remove comments?

Reply:
Changes from master, probably we need it